deploy/cr.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: "konfigurator.stakater.com/v1"
+kind: "ConfigTemplate"
```

Comment:
@waseem-h I wonder if its a good name; I mean will it be same for Secrets? As **Config** feels like for just **ConfigMap**

Reply:
It will be the same for configmap. I thought about creating 2 different CRD's vs 1. I went with 1 because it should be easy to work and won't duplicate any logic in case of 2. We have a renderTarget property that tells where should the templates render i.e., ConfigMap or Secret.

You're right about the name though, what do you suggest? @rasheedamir